src/app/data-planes/components/data-plane-traffic/ServiceTrafficCard.vue

```diff
@@ -3,7 +3,7 @@
     class="service-traffic-card"
   >
     <template #title>
-      <KBadge :appearance="props.protocol === 'unknown' ? 'success' : 'info'">
+      <KBadge :appearance="props.protocol === 'passthrough' ? 'success' : 'info'">
```

Comment:
I think in Kuma we have unknown protocol for internal use, and in case it's unknown we use default which is TCP. Passthrough cluster is a tcp cluster but showing it as `passthorough` is easier for user to understand

Reply:
Ah gotcha, good to know!